mocks/marketplace_api.py

```diff
@@ -43,14 +44,14 @@ def _validate_manifest(self, app):
 
         # validate manifest
         validation_report = self._client.is_manifest_valid(manifest_validation_id)
-        Assert.true(validation_report, "The manifest url is not valid.\n Validation report:\n %s" % validation_report)
+        assert validation_report, 'The manifest url is not valid.\n Validation report:\n %s' % validation_report
```

Comment:
This seems strange. It's just an `assert`, i.e., assert True, so if `validation_report` contained anything other than something that evaluates explicitly as `False`, it would pass. It seems to me that therefore the contents of `validation_report` if this assert fails would be of little use. I wonder if this assert is even correct.